### DISCUSSION TITLE
"For You" shows some old contents

### DISCUSSION THREAD

**Discussion by silverstone1903** [community/community#35098] [Other Feature Feedback, Questions, & Ideas]
Hi 🖐🏻

On my for you page there are some feed elements says that "x has a new discussion in _announcements_ - 150d". I know it is something I haven't seen yet, however it is almost 4 months old. It says that " You're seeing this because you starred x/y" but it's something old and not worth to discover anymore I think.

This is not a problem, still I wanted to share as a feedback!

**Comment by joshistoast** [comment]
Yeah I just got like 3 rcs announcements from almost a year ago each mixed with some announcements made today. Only started happening for me as of recent :/

whack.
